sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -267,4 +267,50 @@ inline bool operator>=(std::chrono::system_clock::time_point const& tp, DateTime
   return (dt <= tp);
 }
 
+namespace _internal {
```

Comment:
I was expecting this be Azure::Core::_internal.

Reply:
Sounds good - I'll moved it to `Core`, and once CI succeeds, I am checking it in.